third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -92,6 +92,10 @@ output "cluster_ca_certificate" {
 * `cluster_ipv4_cidr` - (Optional) The IP address range of the kubernetes pods in
     this cluster. Default is an automatically assigned CIDR.
 
+* `cluster_autoscaling` - (Optional) Configuration for cluster autoscaling, as described
```

Comment:
Hi @ndmckinley ! You refer to node autoprovisioning feature as cluster autoscaling. It is a bit confusing - GKE has both options `--enable-autoscaling` and `--enable-autoprovisioning`. The difference between them is the first operates on existing node pools while the latter creates node pools based on the resources requests.

Can you confirm that this contribution adds `--enable-autoprovisioning` feature to google Terraform provider?